lib/scenario-editor/utils/valhalla.js

```diff
@@ -195,13 +195,21 @@ export async function getSegment (
  * Example URL: https://graphhopper.com/api/1/route?point=49.932707,11.588051&point=50.3404,11.64705&vehicle=car&debug=true&&type=json
  */
 export function routeWithGraphHopper (points: Array<LatLng>): ?Promise<GraphHopperResponse> {
+  var graphhopper_url
```

Comment:
The changes mostly look good @mvanlaar! One note: this could all be done in one (ish) line with a constant:

```javascript
// Use custom url if it exists, otherwise default to the hosted service.
const GRAPH_HOPPER_URL = process.env.GRAPH_HOPPER_URL || 
  'https://graphhopper.com/api/1/'
```

Reply:
Remove this unused declaration `var graphhopper_url`, please.